src/main/java/com/faforever/client/chat/ChatController.java

```diff
@@ -237,7 +237,7 @@ public void onJoinChannelButtonClicked() {
     channelNameTextField.clear();
     if (!channelName.startsWith("#")) {
       log.info("Channel name {} does not start with #", channelName);
-      notificationService.addImmediateErrorNotification(new IllegalArgumentException(), "chat.error.noHashTag", channelName);
+      notificationService.addImmediateNotification("errorTitle", "chat.error.noHashTag", channelName);
```

Comment:
Please use a constant for "errorTitle"

Reply:
Will probably make an addImmediateWarnNotification function instead